Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:14.04
+
+ADD ./WcaOnRails /app/WcaOnRails
```

Comment:
Having this line at the top causes a rebuild of the entire image every time you change a file which is rather annoying when developing.

There's also a small trick to avoid rerunning `bundle install`: Copy `Gemfile` and `Gemfile.lock` to the container, then run `bundle install` and then copy the rest of the application. This will cache the `bundle install` command and only rerun it if you change the `Gemfile(.lock)`. Here's an example how this might look like: https://coderwall.com/p/i0g_1q/caching-bundler-when-building-a-ruby-rails-app-s-image-with-docker